cranelift-simplejit/src/backend.rs

```diff
@@ -492,13 +510,13 @@ fn lookup_with_dlsym(name: &str) -> *const u8 {
 
         for handle in &handles {
             let addr = winapi::um::libloaderapi::GetProcAddress(*handle, c_str_ptr);
-            if addr.is_null() {
+            if addr.isnull() {
```

Comment:
All three of these `is_null` changes seem wrong to me. Note that these are only in code used for windows builds.

Reply:
Good catch; these are likely from a search-and-replace that happened quite a while ago to rename the wasm instruction, but these aren't talking about the wasm instruction, so we should change these back.

Reply:
Yeah, good catch. The change was carried over from #781. Removed in the latest commit.